yarn.lock

```diff
@@ -2196,6 +2208,11 @@ copy-descriptor@^0.1.0:
   resolved "https://registry.yarnpkg.com/copy-descriptor/-/copy-descriptor-0.1.1.tgz#676f6eb3c39997c2ee1ac3a924fd6124748f578d"
   integrity sha1-Z29us8OZl8LuGsOpJP1hJHSPV40=
 
+core-js@3.1.4:
+  version "3.1.4"
+  resolved "https://registry.yarnpkg.com/core-js/-/core-js-3.1.4.tgz#3a2837fc48e582e1ae25907afcd6cf03b0cc7a07"
+  integrity sha512-YNZN8lt82XIMLnLirj9MhKDFZHalwzzrL9YLt6eb0T5D0EDl4IQ90IGkua8mHbnxNrkj1d8hbdizMc0Qmg1WnQ==
+
```

Comment:
SimpleBar already pulls in core-js 3.0.1 as `^3.0.1`. Can we convince it to use 3.1.4 too?

(And I’d prefer `core-js@^3.1.4` over `core-js@3.1.4` so as to avoid reintroducing a similar issue later.)

Reply:
I think we can do something like https://github.com/zulip/zulip/blob/master/package.json#L87 again.